draft-ietf-tls-tls13.md

```diff
@@ -3331,7 +3355,7 @@ useful as a traffic analysis countermeasure.
 
        struct {
            ContentType type;
-           ProtocolVersion legacy_record_version = 0x0301;    /* TLS v1.x */
+           ProtocolVersion legacy_record_version;
```

Comment:
The argument is roughly: if we leave it in, a TLS 1.3 server is justified in enforcing the literal bytes and throwing a decode_error if they don't match.  But that means that it will choke hard on (e.g.) certain TLS 1.1 ClientHellos, when it should really be sending an abort noting that the versions don't match (as appropriate).  At least EKR and davidben were in support of the position that you should not be checking these bytes prior to encryption.